### ISSUE TITLE
libde265 frame-parallel branch still recommended?

### ISSUE THREAD

**Issue by jonpovey** [strukturag/libheif#807]
Hi, I am taking a first look at libheif and libde265. I noticed that the README for this project recommends the `frame-parallel` branch of libde265, which has a tip commit from 2019, where `master` has been worked on recently.

Just wondered if the README for this project needs updating. Thanks

**Comment by farindk** [comment]
Right. Actually, 'frame-parallel' should have become the new master branch because it had faster (video) decoding. For still images, there is probably no performance difference. Thus, it seems that it got neglected and things have progressed so far that it's difficult to use the frame-parallel branch at all. I have removed the recommendation from the README.

But I'll leave this issue open since we still need to resolve this is some way in the future.

**Comment by bradh** [comment]
Should this issue be against https://github.com/strukturag/libde265 ? Doesn't seem like it can be actioned in libheif.

**Comment by farindk** [comment]
@bradh It's affecting both, because libheif also has special code for the "new" libde265 API. I don't know yet how to proceed here. It's also not that urgent.
